com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
+Path tracing changes the way refraction models behave. The table below describes how each refraction model behaves when you enable path tracing. To change the type of refraction model a Lit Material uses, set its **Surface type** to **Transparent** then select a method from the **Refraction model** drop down.
 
-| Refraction model  | Path tracing meaning                              | Surface sidedness                                 |
-|-------------------|---------------------------------------------------|---------------------------------------------------|
-| *Box* or *Sphere* | *Thick* object (e.g magnifying paperweight)       | Double sided, with *None* normal mode             |
-| *Thin*            | *Thin* object (e.g soap bubble or window)         | Double sided, with *Flip* or *Mirror* normal mode |
-| *None*            | *Thin* object, with smoothness = 1 and no Fresnel | Double sided, with *Flip* or *Mirror* normal mode |
+| Refraction model | Path tracing behavior                                        | Compatible Surface sides                                     |
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Box**          | A box-shaped model where incident light enters through a flat surface and leaves through a flat surface.<br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Sphere**       | A sphere-shaped model that produces a magnifying glass-like effect to refraction. <br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Thin**         | A thin box surface type, similar to **Box** with a fixed thickness of 5cm. Select this for thin window-like surfaces. <br/>The behaviour of the **Thin** refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a window. | A double-sided Material that has its **Normal mode** set to  **Flip** or **Mirror**. |
+| **None**         | A thin refractive surface, hardcoded to be fully smooth to simulate alpha blending.<br/>The behaviour of the **None **refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a window. | A double-sided Material that has its **Normal mode** set to  **Flip** or **Mirror**. |
```

Comment:
I wouldn't say that this mode is suitable for anything really, it's just here to keep compatibility with raster mode when you switch to path tracing, but other than that **thin** should be prefered in every single situation.